controller/pb/controller.proto

```diff
@@ -200,6 +200,7 @@ message InvokeConfig {
   bool Tty = 8;
   bool Rollback = 9; // Kill all process in the container and recreate it.
   bool Initial = 10; // Run container from the initial state of that stage (supported only on the failed step)
+  bool NoCmd = 11; // Do not set cmd but use the image's default
```

Comment:
Can we put this after the `Cmd` field? We can keep the number the same, but would be good to group them together.